accessibility.md

```diff
@@ -60,6 +60,20 @@ For all products, test changes using a screen magnifier:
 - [Zoom on OSX](https://support.apple.com/en-gb/guide/mac-help/mh40579/10.15/mac/10.15)
 - [Magnifier on Windows](https://support.microsoft.com/en-gb/help/11542/windows-use-magnifier-to-make-things-easier-to-see)
 
+## Design system
+
+In web products, use the standard [UI components][ui-components] and [typography snippets][typography-snippets] provided by the Source Design System. These are optimised for accessibility.
```

Comment:
Maybe another link?

Love your new style of links ;)

Reply:
Oh cool, much prettier. Thanks @mxdvl!